build-system/pr-check.js

```diff
@@ -16,33 +16,10 @@
 'use strict';
 
 const {APPS_TO_TEST, determineBuildTargets} = require('./build-targets');
-const {cyan, green} = require('kleur/colors');
-const {execOrDie} = require('./exec');
+const {cyan} = require('kleur/colors');
```

Comment:
```suggestion
const {cyan} = require('./common/colors');
```

Reply:
This is the `amp-github-apps` repo. There's no `common/` directory 😃